api/RTCErrorEvent.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "RTCErrorEvent": {
+        "__compat": {
+          "description": "<code>RTCErrorEvent()</code> constructor",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcerrorevent-constructor",
+          "support": {
+            "chrome": {
+              "version_added": "74"
+            },
+            "chrome_android": {
+              "version_added": "74"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "62"
+            },
+            "opera_android": {
+              "version_added": "53"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": "11.0"
+            },
+            "webview_android": {
+              "version_added": "74"
+            }
+          },
+          "status": {
+            "experimental": false,
```

Comment:
Yep, this is correct per https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status since Chrome 74 was released on 2019-04-23.